test/pointing_party/vote_calculator_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule PointingParty.VoteCalculatorTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+
+  describe "calculate_votes/1" do
+    setup do
+      points_map = StreamData.fixed_map(%{points: integer(1..5)})
+      metas_map  = StreamData.fixed_map(%{
```

Comment:
`use ExUnitProperties` will import `StreamData` functions. We can omit the `StreamData` prefix here.